api/ccapi/include/layer.h

```diff
@@ -71,6 +71,8 @@ enum LayerType {
   LAYER_RESHAPE,                           /**< Reshape Layer type */
   LAYER_RNNCELL,                           /**< RNN Cell Layer type */
   LAYER_LSTMCELL,                          /**< LSTM Cell Layer type */
+  LAYER_LSTMCELLCORE,                      /**< LSTM Cell Core Layer type */
```

Comment:
For later) I think core shouldn't be exposed, if allow two input kind of change, this will be done changing `LAYER_LSTMCELL`

Reply:
Done